src/task/MonitorTask.ts

```diff
@@ -283,14 +293,11 @@ export default class MonitorTask
       }
     }
     if (fireCallback) {
-      this.logger.debug(() => {
-        return `Downlink video streams are not receiving enough data`;
-      });
-      this.context.audioVideoController.forEachObserver((observer: AudioVideoObserver) => {
-        Maybe.of(observer.videoNotReceivingEnoughData).map(f =>
-          f.bind(observer)(Array.from(videoReceivingBitrateMap.values()))
-        );
-      });
+      this.logger.info(
+        `One or more video streams are not receiving expected amounts of data ${JSON.stringify(
```

Comment:
Can we loosen these threshold while we are here? currently its a bit noisy. Maybe .2 and 2.0

Reply:
Updated